src/components/profile_image.tsx

```diff
@@ -15,15 +15,13 @@ export interface ProfileProps {
 export const ProfileImage: React.StatelessComponent<ProfileProps> = ({ size, imageURL }) => (
     <div className="profile-image">
         {imageURL
-            && <Image src={imageURL} size={size} />
-        }
+            && <Image src={imageURL} size={size} />}
         {!imageURL
             && (
                 <CircleIcon appearance="primary" disabled size={size}>
                     <ProfileIcon />
                 </CircleIcon>
-            )
-        }
+            )}
```

Comment:
Just to check: we want `}` to follow the closing tag, but `)}` can be in a separate line...?

Reply:
🤷‍♂ I don't really care one way or another but this is what AirBnB's style guide says to do.

Reply:
Oh I know why. It's because they want the closing curly bracket to be next to whatever is the end of the expression. Sometimes that's the tag, but if there's a conditional that might mean there's the parenthesis.